### DISCUSSION TITLE
[BUG]部分邮件点击后，整体布局会被改变

### DISCUSSION THREAD

**Discussion by LiJunYi2** [dreamhunter2333/cloudflare_temp_email#491] [General]
## 复现步骤
部分邮件内容会导致整体布局被改变，可直接查看 `https://mail.awsl.uk/` 示例下 `tmpradleylickauch94@awsl.uk` 账号邮件。

<img width="832" alt="image" src="https://github.com/user-attachments/assets/36358274-2875-4ce9-a0aa-6c182c4e58e4">

## 预期行为
查看邮件内容不应该改变整体布局

## 部署方式

- cli 部署

## 浏览器环境
google 版本 130.0.6723.70

**Comment by dreamhunter2333** [comment]
暂时解决不了邮件 html css的问题，可以在主页账户设置中切换成 「使用iframe显示HTML邮件」
